imitation_nav_node:
  ros__parameters:
    interval_ms: 100
    localization_interval_ms: 500
    model_name: model_fuctory.pt
    image_height: 88
    image_width: 200
    max_linear_vel: 1.0
    max_angular_vel: 1.57
    visualize_flag: true